src/tests/ftest/util/ec_utils.py

```diff
@@ -413,56 +414,55 @@ def start_online_single_operation(self, operation, parity=1):
 class ErasureCodeMdtest(MdtestBase):
     """Class to used for EC testing for MDtest Benchmark."""
 
-    def __init__(self, *args, **kwargs):
-        """Initialize a MdtestBase object."""
-        super().__init__(*args, **kwargs)
-        self.server_count = None
-        self.set_online_rebuild = False
-        self.rank_to_kill = None
-        self.obj_class = None
-
     def setUp(self):
         """Set up each test case."""
         super().setUp()
-        engine_count = self.server_managers[0].get_config_value("engines_per_host")
-        self.server_count = len(self.hostlist_servers) * engine_count
-        self.obj_class = self.params.get("dfs_oclass_list", '/run/mdtest/objectclass/*')
         # Create Pool
         self.add_pool()
+        self.container = None
         self.out_queue = queue.Queue()
 
-    def write_single_mdtest_dataset(self):
-        """Run MDtest with EC object type."""
-        # Update the MDtest obj class
-        self.mdtest_cmd.dfs_oclass.update(self.obj_class)
+    def _start_execute_mdtest(self, mdtest_result_queue):
+        """Run the execute_mdtest method
 
-        # Write the MDtest data
-        self.execute_mdtest(self.out_queue)
+        Args:
+            mdtest_result_queue(Queue) : Queue for passing errors.
+        Returns:
+            result(object) : mdtest run result
+        """
+        try:
+            result = self.execute_mdtest(mdtest_result_queue)
+        except (CommandFailure, DaosApiError, DaosTestError):
```

Comment:
We need to catch all exceptions because we don't know what might go wrong
```suggestion
        except Exception:  # pylint: disable=broad-except
```

Reply:
Will update it.

Reply:
done.